packages/core/src/SpringRef.ts

```diff
@@ -7,9 +7,28 @@ interface ControllerUpdateFn<State extends Lookup = Lookup> {
   (i: number, ctrl: Controller<State>): ControllerUpdate<State> | Falsy
 }
 
-export class SpringRef<State extends Lookup = Lookup> {
+/**
+ * Extending from function allows SpringRef instances to be callable.
+ * https://hackernoon.com/creating-callable-objects-in-javascript-d21l3te1
+ *
+ * ```js
+ * const [springs, animate] = useSpring(() => ({}))
+ * animate.set({}) // this works
```

Comment:
this should probably reflect what we discussed re API.

Reply:
Done, I'll let you resolve this!